tests/test_public_api.py

```diff
@@ -80,11 +80,17 @@ def test_parse_path_user_proj(mock_server, api):
     assert r == "run"
 
 
-def test_parse_path_proj(mock_server, api):
-    u, p, r = api._parse_path("proj")
+def test_parse_path_run(mock_server, api):
```

Comment:
Hey @martinabeleda, we recently moved our unit tests to `tests/unit_tests` -- could you please resolve the conflict? thanks!